src/saml.ts

```diff
@@ -1408,7 +1408,12 @@ class SAML {
       "@Binding": "urn:oasis:names:tc:SAML:2.0:bindings:HTTP-POST",
       "@Location": this.getCallbackUrl(),
     };
-    return buildXmlBuilderObject(metadata, true);
+
+    let metadataXml = buildXmlBuilderObject(metadata, true);
+    if (this.options.signMetadata && isValidSamlSigningOptions(this.options)) {
```

Comment:
Likewise, lets just test `=== true` as that better matches the types.

Reply:
👍